### ISSUE TITLE
dump files are not created in /opt/intelmq/var/log

### ISSUE THREAD

**Issue by aaronkaplan** [certtools/intelmq#686]
Since some time it seems that the dump files are created in /opt/intelmq/var/

```bash
root@intelmq-dev2:/opt/intelmq/var# pwd
/opt/intelmq/var

root@intelmq-dev2:/opt/intelmq/var# ls -al *snmp*
-rw-r--r-- 1 intelmq intelmq 1686851 Sep 17 12:52 logopensnmp-parser.dump

# ... trying to read it...

root@intelmq-dev2:/opt/intelmq/var# sudo -Hu intelmq /usr/local/bin/intelmqdump logopensnmp-parser
Given file does not exist: /opt/intelmq/var/log/logopensnmp-parser.dump
```

It looks like a "/" was forgotten somewhere.

**Comment by sebix** [comment]
> It looks like a "/" was forgotten somewhere.

yes, in the configuration :D But fixed in 895564b96b8e14a3d93bda756d26444e18def577 to ensure this never happens again!

**Comment by aaronkaplan** [comment]
> On 19 Sep 2016, at 14:04, Sebastian <notifications@github.com> wrote:
> 
> It looks like a "/" was forgotten somewhere.
> 
> yes, in the configuration :D But fixed in 895564b to ensure this never happens again!
> 

Thanks, excellent!
